examples/ddpg_pmsm_dq_current_control.py

```diff
@@ -144,24 +209,30 @@ def reset(self, **kwargs):
         size=2
     )
 
-    # Create the agent
+    # Create the agent for DDPG learning
     agent = DDPGAgent(
+        # Pass the previously defined characteristics
         nb_actions=nb_actions,
         actor=actor,
         critic=critic,
         critic_action_input=action_input,
         memory=memory,
         random_process=random_process,
+
+        # Define the overall training parameters
         nb_steps_warmup_actor=2048,
         nb_steps_warmup_critic=1024,
         target_model_update=1000,
         gamma=0.9,
         batch_size=128,
         memory_interval=2
     )
+
+    # Compile the models within the agent (making them ready for training)
+    # Note that the DDPG agent uses two models, hence we define two optimizers here

```

Comment:
I would not call them "models" -> better "... two function approximators, hence..."